grunt_tasks/server.js

```diff
@@ -44,7 +44,8 @@ module.exports = function (grunt) {
     grunt.config.merge({
         shell: {
             readServerConfig: {
-                command: 'env python config_parse.py girder/conf/girder.local.cfg',
+                command: 'env python config_parse.py ' +
+                         grunt.config.get('girderDir') + '/conf/girder.local.cfg',
```

Comment:
Is there any particular reason that the api and static roots can't just be assumed to be the defaults in [`testEnv.jadehtml`](https://github.com/girder/girder/blob/master/clients/web/test/testEnv.jadehtml#L13)?  If so, I think the task can be removed completely.  If those values do need to be customized, maybe it is something that can be moved to the CMake config as it is only relevant for testing.